frontend/tests/controllers-test/changePwdCtrl.test.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+describe('Unit tests for Change PWD controller', function () {
```

Comment:
`Unit tests for Change PWD controller` -> `Unit tests for change password controller`

Reply:
@RishabhJain2018 done